src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist

```diff
@@ -577,6 +585,8 @@ staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods minus java.util.Co
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods minus java.util.List java.lang.Iterable
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods minus java.util.List java.lang.Object
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods minus java.util.List java.util.Collection
+staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods minus java.util.List removeAt int
+staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods minus java.util.List removeLast
```

Comment:
@haridsv These entries are malformed. `removeAt` and `removeLast` aren't types, what methods were you trying to whitelist? See https://github.com/jenkinsci/script-security-plugin/pull/268.

Reply:
I think my intention was to whitelist these two methods: http://docs.groovy-lang.org/latest/html/groovy-jdk/java/util/List.html#removeAt(int) and http://docs.groovy-lang.org/latest/html/groovy-jdk/java/util/List.html#removeLast() so I seem to have started with wrong lines and not realize because there was no test failure :(

I think the right entries would have been:

```
staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods removeAll java.util.Collection groovy.lang.Closure
staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods removeLast java.util.List
```

However, the last one is only available in Groovy 2.5.0, so can't work. In case you don't mind taking this change forward via you PR, I will add a suggestion in your other RP.